Documentation/Internal/Grafana/AzureMonitorContainers-GenevaLogs_Multitenancy-Platform.json

```diff
@@ -0,0 +1,1134 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
```

Comment:
please double check that uid is this format

Reply:
> please double check that uid is this format

This is correct. This is generated by Grafana tool.